src/pkg/utils/ubuntu.go

```diff
@@ -38,6 +38,15 @@ func getFullyQualifiedImageUbuntu(image, release string) string {
 	return imageFull
 }
 
+func getP11KitClientPathsUbuntu() []string {
+	paths := []string{
+		"/usr/lib/aarch64-linux-gnu/pkcs11/p11-kit-client.so",
+		"/usr/lib/x86_64-linux-gnu/pkcs11/p11-kit-client.so",
```

Comment:
@Jmennius I hope I got the path to `p11-kit-client.so` right for Ubuntu. I suppose we build the `ubuntu-toolbox` images only for `aarch64` and `x86_64`, correct?